### PR TITLE
Fix regex and xmldata BBEs output verification failure

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subect

### CODE REVIEW

File: ballerina/build.gradle [hunk 1]
```diff
@@ -877,7 +877,8 @@ task testExamples() {
             lines.each {
                 str = it.trim()
                 if (str != "Compiling source" && str != "${balName}.bal" && str != "Running executable" && str != ""
-                        && str != "Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8" && !str.startsWith('HINT')) {
+                        && str != "Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8" && !str.startsWith('HINT') &&
+                        !str.contains("ballerina/")) {
```
Comment (gimantha) [inline ballerina/build.gradle:881]: Shall we put a comment saying why we had to add this line?

Comment (azinneera) [reply ballerina/build.gradle:882]: Wouldn't using `pulled from central successfully` to skip be better?

Comment (kalaiyarasiganeshalingam) [reply ballerina/build.gradle:882]: We have to skip the following logs
`ballerina/xmldata:2.2.2 [central.ballerina.io -> home repo]  100% [==========]`
`ballerina/xmldata:2.2.2 pulled from central successfully`
They both have the same value as `ballerina/`. So I used this.

**Review by gimantha — APPROVED** [review]
LGTM

**Pull request merged**